packages/coil-extension/src/popup/components/AmountInput.tsx

```diff
@@ -195,7 +195,10 @@ export const AmountInput = (): React.ReactElement => {
         <Amount
           size={displayFontSize}
           onClick={() => setIsUserInput(true)}
-          disabled={maxAllowableTipAmountUsd == 0}
+          disabled={
+            maxAllowableTipAmountUsd == 0 ||
+            maxAllowableTipAmountUsd < minTipLimitAmountUsd
```

Comment:
do these map to the same case?  can minTipLimitAmountUsd be 0? (does that even make sense?)

Reply:
I think you're right in that  "maxAllowableTipAmountUsd < minTipLimitAmountUsd" will handle all "maxAllowableTipAmountUsd == 0" but the redundancy does make it a bit more readable? 

I wondered if this calculation should be added to the `calculateMaxAllowableTip` but then I guess it makes what's happening in TipAmountFeedback a bit trickier.